package/boot/uboot-sunxi/patches/300-force-pylibfdt-build.patch

```diff
@@ -0,0 +1,30 @@
+--- a/Makefile
++++ b/Makefile
+@@ -2000,26 +2000,7 @@ endif
+ # Check dtc and pylibfdt, if DTC is provided, else build them
+ PHONY += scripts_dtc
+ scripts_dtc: scripts_basic
+-	$(Q)if test "$(DTC)" = "$(DTC_INTREE)"; then \
```

Comment:
Please add a comment to the patch why this is needed.

Reply:
Same as in uboot-mediatek.